package.json

```diff
@@ -42,12 +42,14 @@
   },
   "dependencies": {
```

Comment:
@wrola what about the rest? Can't they rely on whatever is used by services?

Reply:
I thought the goal is, to have the strong dependencies required as this packages strongly depend on them and are directly use in codebase, the rest of packages is use indirectly so that's way we keep them as peerDependencies?

Reply:
It is "strong" - it just won't add up to packages dependency graph, only one copy (specified by the service) will be added.
From what I see only `ajv` is "strong" in a sense that it may not be included by some services (it's very specific to the logic of this library), so it's a dependency, all `nest` packages are already specified in their required versions by services. Same for class-transformer and class-validation.

Reply:
Ok, I will keep `ajv` , the rest will be moved into `peerDependencies`